File: music-streaming-ui/src/types/album.ts
export interface Album {
  albumId: number;
  title: string;
  artistId: number;
  artistName: string;
  coverImageUrl: string;
  releaseYear: number;
}

export interface CreateAlbumPayload {
  title: string;
  artistId: number;
  releaseYear: number;
  coverImage: File;
}

File: MusicStreamingSystem.API/MusicStreamingSystem.Core/Models/Album.cs
namespace MusicStreamingSystem.Core.Models
{
    public class Album
    {
        public int AlbumId { get; set; }

        public string Title { get; set; } = string.Empty;

        public int ArtistId { get; set; }

        public string CoverImageUrl { get; set; } = string.Empty;

        public int ReleaseYear { get; set; }
    }
}